pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -52,30 +52,51 @@ class RandInt(Module):
 
   def create_returnn_layer_dict(self, low, high, size, dtype=None) -> Dict[str, Any]:
     dtype = dtype or "int64"
-    assert isinstance(low, int)
-    assert isinstance(high, int)
-    assert all(isinstance(sz, int) for sz in size)
-    return {"class": "rand_int", "shape": size, "maxval": high, "minval": low, "dtype": dtype}
+    if isinstance(low, Tensor):
+      low = low.type(dtype)
+      low = self._get_input_layer_name(low)
```

Comment:
Remove this, and use `tensor_entry.is_dim` instead.

Reply:
The `RandIntLayer` does not accept `Dim` for `maxval` or `minval`. Not sure if this would be helpful.

Reply:
Ah sorry, I confused that here.